src/unix/linux_like/android/mod.rs

```diff
@@ -517,6 +517,11 @@ s! {
         pub ifr6_prefixlen: u32,
         pub ifr6_ifindex: c_int,
     }
+
+    // is __be16 __bitwise __u16
+    pub struct __c_anonymous___be16 {
```

Comment:
Why `__c_anonymous_`? This isn't an anonymous type, it's name is `__be16`.

Unless I'm mistaken, `__c_anonymous_fieldname` is normally used for inline structs/unions attached to a field of that name - this is a named type, and that's not the field name.